Treasurer.php

```diff
@@ -416,6 +419,126 @@ function s()
 		echo "</table>";
 	else
 	   echo "<p class='indent1'>There were no trial flights for this month</p>";
+   
+    /*  ---------------
```

Comment:
I think this chunk of code is duplicated 4 times or so. Do you thing we could extract it into a renderer that given a list of flights and columns it renders what is needed?

e.g. the computation of `$towcost` is duplicated 4 times.